setup.py

```diff
@@ -22,13 +22,13 @@
     "Django>=4.2,<6.0",
     "django-modelcluster>=6.2.1,<7.0",
     "django-permissionedforms>=0.1,<1.0",
-    "django-taggit>=5.0,<6.2",
+    "django-taggit>=5.0,<7",
     "django-treebeard>=4.5.1,<5.0",
     "djangorestframework>=3.15.1,<4.0",
     "django-filter>=23.3",
     "draftjs_exporter>=2.1.5,<6.0",
     "Pillow>=9.1.0,<12.0.0",
-    "beautifulsoup4>=4.8,<4.13",
+    "beautifulsoup4>=4.8,<5",
```

Comment:
I doubt they'll be releasing version 5 under the `beautifulsoup4` package name, but there's no harm in adding this 😄